doc/manual.asciidoc

```diff
@@ -265,6 +265,37 @@ output files are out of date.
 rebuild those targets.
 _Available since Ninja 1.11._
 
+`multi-inputs`:: print one or more sets of inputs required to build targets.
+Each line will consist of a target, a delimiter, an input and a terminator character.
+The list produced by the tool can be helpful if one would like to know which targets
+that are affected by a certain input.
++
+The output will be a series of lines with the following elements:
++
+----
+<target> <delimiter> <input> <terminator>
+----
```

Comment:
nit: Add a line stating that by default `<delimiter>` is a single TAB character, and `<terminator>` is a line terminator (i.e. `\n` on Posix, and `\r\n` on Windows), but that flags can be used to change their values.

Reply:
Updated